storage/innobase/log/log0recv.cc

```diff
@@ -2878,7 +2876,7 @@ static bool recv_multi_rec(byte *ptr, byte *end_ptr) {
     space_id_t space_id = 0;
 
     ulint len = recv_parse_log_rec(&type, ptr, end_ptr, &space_id, &page_no,
-                                   true, &body);
+                                   false, &body);
```

Comment:
am thinking if we should get rid of this 'apply' altogether. It can be inferred from block pointer. If block is nullptr, it is scan/parse phase. During apply, we always have block as non null ptr

Reply:
Yes, but I like apply better - it shows some intentions. If I was to reverse engineer this without apply variable, it would have been much harder.

Reply:
done